sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/utils.ts

```diff
@@ -20,23 +20,34 @@ import {
 } from "@opentelemetry/semantic-conventions";
 import { SDK_INFO, hrTimeToMilliseconds } from "@opentelemetry/core";
 import { DataPointType, Histogram, ResourceMetrics } from "@opentelemetry/sdk-metrics";
-import { AZURE_MONITOR_OPENTELEMETRY_VERSION } from "../../types";
+import {
+  AZURE_MONITOR_AUTO_ATTACH,
+  AZURE_MONITOR_OPENTELEMETRY_VERSION,
+  AZURE_MONITOR_PREFIX,
+} from "../../types";
 import { Resource } from "@opentelemetry/resources";
 import { QuickPulseMetricNames, QuickPulseOpenTelemetryMetricNames } from "./types";
+import { getOsPrefix } from "../../utils/common";
+import { getResourceProvider } from "../../utils/common";
 
+/** Get the internal SDK version */
 export function getSdkVersion(): string {
-  const { node } = process.versions;
-  const nodeVersion = node.split(".");
+  const { nodeVersion } = process.versions;
   const opentelemetryVersion = SDK_INFO[SemanticResourceAttributes.TELEMETRY_SDK_VERSION];
-
-  const prefix = process.env["AZURE_MONITOR_AGENT_PREFIX"]
-    ? process.env["AZURE_MONITOR_AGENT_PREFIX"]
-    : "";
   const version = `dst${AZURE_MONITOR_OPENTELEMETRY_VERSION}`;
```

Comment:
Please update to `ext${AZURE_MONITOR_OPENTELEMETRY_VERSION}`;

Reply:
I assume you mean `ext${<exporter version here>}` or are you ok with unifying the distro version over the exporter version? I checked the spec, but unifying the extension version on distro or exporter seems to be an open question.

Reply:
no, I mean using distro version here, we should use exporter version only when distro is not present. We can change this if needed when spec is updated

Reply:
Ah, I see, that makes more sense. I'll align with that.